examples/AlfaKit/Sample/config.h

```diff
@@ -27,29 +27,48 @@
 #define MQTT_BROKER_HOST "mqtt.example.com"
 // MQTT_BROKER_PORT generally is 1883 for clear-text, 8883 for TLS
 #define MQTT_BROKER_PORT 1883
-#define MQTT_KEEP_ALIVE 0
+// Keep-Alive parameter for your connection. Depends on your network conditions,
+//   20 is norlally a safe bet
+#define MQTT_KEEP_ALIVE 20
+
+// Client ID identifies your device on the broker
```

Comment:
I would stress in the comment here again that it must be unique amongst connected clients